dotnet/src/Connectors/Connectors.MistralAI/MistralChatMessageContent.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI;
+
+/// <summary>
+/// Mistral specialized chat message content
+/// </summary>
+public sealed class MistralChatMessageContent : ChatMessageContent
```

Comment:
Not sure I understand the purpose of this class, can we just return `ChatMessageContent`? It doesn't look like it has any differences with `ChatMessageContent` except naming.